src/app/organization/tellers/tellers.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Teller
+  </button>
+
+</div>
+
+<div class="container">
+
+  <div fxLayout="row" fxLayoutGap="20px">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="officeName">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Branch </th>
+        <td mat-cell *matCellDef="let teller"> {{ teller.officeName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Teller Name </th>
+        <td mat-cell *matCellDef="let teller"> {{ teller.name }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let teller">
+          <div [className]="teller.status === 'ACTIVE' ? 'true' : 'false'">
+            <fa-icon matTooltip="{{ teller.status === 'ACTIVE' ? 'Active' : 'Inactive' }}" matTooltipPosition="right" icon="circle" size="lg"></fa-icon>
+          </div>    
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="startDate">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Started On </th>
+        <td mat-cell *matCellDef="let teller"> {{ teller.startDate | date }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="viewCashiers">
+        <th mat-header-cell *matHeaderCellDef></th>
+        <td mat-cell *matCellDef="let teller">
+          <button mat-button color="primary">
+            <fa-icon icon="eye"></fa-icon>&nbsp;&nbsp;View Cashiers
+          </button>
+        </td>
+      </ng-container>
```

Comment:
Have a look at Accounting -> Provisioning Entries and implement this in a similar manner. Add a heading to the column and change the icon.

Reply:
1. I've added a header
2. I kept the button as 'View Cashiers' rather than 'View' like provisioning entries as it would be quite confusing and would look like it is a button to view the teller even with the heading.
3. I am not sure what other icon I should be using. I used faEye to keep consistency with other components. Should this be faList instead?

Reply:
No, the button is not supposed to be disabled here. I was just talking about the heading and the icon. Actually, the icon was fine I confused myself with something else. My apologies.